### PR TITLE
React Native version update

### PR DESCRIPTION
**Pull request by Safi-Ullah**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,9 +1,8 @@
 {
-  "main": "node_modules/expo/AppEntry.js",
   "scripts": {
-    "start": "expo start",
```
Comment (fenske) [inline package.json:4]: Did you manage to spin it up locally on the updated lib versions?

File: package.json [hunk 2]
```diff
@@ -17,29 +16,32 @@
...
-    "react-native-gesture-handler": "^2.4.1",
-    "react-native-safe-area-context": "^4.2.4",
-    "react-native-screens": "^3.13.1",
-    "react-native-web": "^0.17.7"
+    "expo-cli": "^6.1.0",
+    "expo-constants": "~14.0.2",
+    "expo-font": "~11.0.1",
+    "expo-linking": "~3.3.0",
+    "expo-splash-screen": "~0.17.5",
+    "expo-status-bar": "~1.4.2",
```
Comment (fenske) [inline package.json:26]: Why transition from ^ to ~ ?

**Review by fenske — COMMENTED** [review]
Thanks for the PR! Left a couple of comments.

File: package.json (hunk 1 above)
Comment (Safi-Ullah) [reply package.json:4]: Yes, I have verified, and the updated version of the app is working fine.

File: package.json (hunk 2 above)
Comment (Safi-Ullah) [reply package.json]: They must've updated with react library update. I've reverted ~ to ^.

**Pull request merged**